### PR TITLE
#683. QnReferredTo can answer if rultor is mentions in comment.

### PR DESCRIPTION
**Pull request by alevohin**
#683. QnReferredTo can answer if rultor is mentions in comment.

### CODE REVIEW

**Comment by alex-palevsky** [comment]
Thanks, I'll ask someone to do a code review here, thanks for your pull request

**Comment by alex-palevsky** [comment]
@krzyk review this please

File: src/test/java/com/rultor/agents/github/qtn/QnReferredToTest.java [hunk 1]
```diff
@@ -72,4 +75,30 @@ public void buildsRequest() throws Exception {
...
 
+    /**
+     * QnReferredTo can answer when mention.
+     * @throws Exception In case of error.
+     */
+    @Test
+    public void answerWhenMentioned() throws Exception {
+        final Repo repo = new MkGithub().randomRepo();
+        final Issue issue = repo.issues().create("", "");
+        issue.comments().post("hello @xx deploy");
```
Comment (krzyk) [inline src/test/java/com/rultor/agents/github/qtn/QnReferredToTest.java]: let's move `xx` to a parameter and use it in all the strings below

File: src/main/java/com/rultor/agents/github/qtn/QnReferredTo.java [hunk 2]
```diff
@@ -78,6 +86,15 @@ public Req understand(final Comment.Smart comment,
         final Req req;
         if (comment.body().trim().startsWith(prefix)) {
             req = this.origin.understand(comment, home);
+        } else if (comment.body().trim().contains(prefix)) {
```
Comment (krzyk) [inline src/main/java/com/rultor/agents/github/qtn/QnReferredTo.java]: there is no need to `trim()` when looking for a string

**Comment by krzyk** [comment]
@alevohin two comments above

**Comment by alevohin** [comment]
@krzyk Thanks! Please review one more time.

**Comment by krzyk** [comment]
@rultor merge pls

**Comment by yegor256** [comment]
@rultor merge pls

**Comment by alex-palevsky** [comment]
@rultor deploy

**Comment by alex-palevsky** [comment]
@krzyk Thanks so much! Your account was topped up for **24 mins** (transaction ID is `50266328`)

+24 added to your rating, current score is: [+1623](http://www.netbout.com/b/35035?open=rating)

**Pull request merged**